packages/core/src/components/button/_common.scss

```diff
@@ -184,6 +188,10 @@ $non-default-state-selectors: ":last-child, :hover, :active, .pt-active, [class*
     background-image: none;
     color: $dark-button-color-disabled;
   }
+
+  .pt-button-loading-spinner .pt-spinner-head {
+    stroke: $pt-dark-text-color;
```

Comment:
For these values, we may be able to reuse some color variables from https://github.com/palantir/blueprint/blob/master/packages/core/src/components/spinner/_spinner.scss